src/table/table.scss

```diff
@@ -122,265 +251,316 @@
         }
       }
 
-      &.iui-tables-head {
-        @include themed {
-          background-color: t(iui-color-background-3);
-        }
-        user-select: none;
+      // #region Indented rows
+      &.iui-expander-indent-0 {
+        padding-left: 0;
+      }
 
-        .iui-icon {
-          @include themed {
-            fill: t(iui-icons-color-actionable);
-          }
+      &.iui-expander-indent-1 {
+        padding-left: $iui-xl;
+      }
 
-          &:hover {
-            @include themed {
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+      &.iui-expander-indent-2 {
+        padding-left: $iui-xl * 2;
+      }
 
-        > .iui-filter {
-          margin-left: $iui-sm;
-          cursor: pointer;
+      &.iui-indent-1 {
+        padding-left: $iui-xl + $iui-s;
+      }
 
-          > .iui-icon {
-            width: $iui-icons-default;
-            height: $iui-icons-default;
-          }
-        }
+      &.iui-indent-2 {
+        padding-left: ($iui-xl + $iui-s) * 2;
+      }
+      // #endregion Indented rows
+    }
 
-        > .iui-sort {
-          flex-grow: 1;
-          justify-content: flex-end !important;
+    // #region Selection
+    &:not(.iui-selected) + .iui-selected {
+      padding-bottom: $iui-xxs;
+      border-bottom: none;
+    }
 
-          > .iui-icon-wrapper {
-            cursor: pointer;
+    &.iui-selected {
+      padding: 0;
+      border-bottom: none;
+      @include themed {
+        border: solid 1px t(iui-color-foreground-primary);
+        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+      }
 
-            > .iui-icon {
-              width: $iui-icons-default;
-              height: $iui-icons-default;
-            }
-          }
+      &:hover {
+        @include themed {
+          background: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-5)
+          );
         }
+      }
 
-        > .iui-filter,
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          height: $iui-icons-default;
-          width: $iui-icons-default;
-          display: flex;
-          justify-content: center;
-          align-items: center;
-          visibility: hidden;
+      + .iui-selected {
+        border-bottom: none;
+        padding-bottom: $iui-xxs;
+        padding-top: 0;
+        @include themed {
+          border-top: 1px solid
+            rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
         }
+      }
 
-        > .iui-filter.iui-active {
-          visibility: visible;
-  
-          > .iui-icon {
-            @include themed {
-              fill: t(iui-icons-color-primary);
-            }
-          }
+      &:last-child {
+        padding-bottom: 1px;
```

Comment:
hmm..this looks weird, but I see the diff.. Need to check deeper, from what it appears and if we could solve it differently..

Reply:
If you can do something better, I would be happy, but considering the limitation of CSS, this was the best I could do.